Exit code: 1
---
Stdout:
======================= Schemathesis test session starts =======================
Schema location: http://127.0.0.1/schema.yaml
Base URL: http://127.0.0.1/api
Specification version: Open API 3.0.2
Random seed: 42
Workers: 1
Collected API operations: 1
Collected API links: 0
API probing: SUCCESS

GET /api/success E                                                        [100%]

==================================== ERRORS ====================================
_______________________________ GET /api/success _______________________________
Runtime Error

division by zero

    Traceback (most recent call last):
      File "/package-root/runner/phases/unit/_executor.py", line XXX, in wrapped
        f(ctx=ctx, case=case, result=result, **kwargs)
      File "/package-root/runner/phases/unit/_executor.py", line XXX, in test_func
        validate_response(
      File "/package-root/runner/phases/unit/_executor.py", line XXX, in validate_response
        failures = run_checks(
      File "/package-root/checks.py", line XXX, in run_checks
        skip_check = check(ctx, response, case)
      File "/tmp/module.py", line XXX, in with_error
        1 / 0
    ZeroDivisionError: division by zero

Need more help?
    Join our Discord server: https://discord.gg/R9ASRAmHnA
=================================== SUMMARY ====================================

No checks were performed.

============================== 1 errored in 1.00s ==============================
